service/create/smallcloudconfig.go

```diff
@@ -0,0 +1,41 @@
+package create
+
+import (
+	"bytes"
+	"encoding/base64"
+	"html/template"
+	"io/ioutil"
+
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+var (
+	smallCloudconfigPath = "templates/user-data.sh"
+)
+
+type SmallCloudconfigConfig struct {
+	MachineType string
+	Region      string
+	S3Bucket    string
+	ClusterID   string
+}
+
+func (s *Service) SmallCloudconfig(config SmallCloudconfigConfig) (string, error) {
+	b, err := ioutil.ReadFile(smallCloudconfigPath)
```

Comment:
I forgot about this because I was focusing on getting something working quick. I'll re-work this.